boards/itsybitsy_m4/examples/spi.rs

```diff
@@ -0,0 +1,73 @@
+#![no_std]
+#![no_main]
+
+/// This example shows a simple transfer operation with an slave device.
+/// The ItsyBitsy will send a simple Hello World message, and the slave
+/// is expected to send a response. After the transaction, the response
+/// from the slave is echoed in the default UART.
```

Comment:
I think the module level docs use `//!` instead of `///`. Not sure this **_actually_** matters, just calling it out